packages/rest-typings/src/v1/moderation/ModerationReportUserPOST.ts

```diff
@@ -0,0 +1,25 @@
+import type { IUser } from '@rocket.chat/core-typings';
+
+import { ajv } from '../Ajv';
+
+export type ModerationReportUserPOST = {
+	userId: IUser['_id'];
+	description: string;
+};
+
+const reportUserPropsSchema = {
+	type: 'object',
+	properties: {
+		userId: {
+			type: 'string',
+		},
+		description: {
+			type: 'string',
+			nullable: true,
+		},
+	},
+	required: ['userId', 'description'],
```

Comment:
If the `description` is optional (nullable), wouldn't be better to remove it as required?

Reply:
Sorry, `description` should actually be required :grimacing: